ethereum/smart-contract/contracts/GatewayToken.sol

```diff
@@ -266,6 +254,19 @@ contract GatewayToken is
         return true;
     }
 
+    /**
+     * @dev Transfers are disabled for Gateway Tokens - override ERC3525 approve functions to revert
+     * Note - transferFrom and safeTransferFrom are disabled indirectly with the _isApprovedOrOwner function
+     */
+    // solhint-disable-next-line no-empty-blocks
+    function approve(uint256, address, uint256) public payable virtual override {
```

Comment:
I still add the argument names here.

Reply:
The linter won't be happy about unused variables.

Reply:
We could disable those? Just noting because the interface generation changed. Happy to leave as is.

Reply:
I'm also happy either way - closing as is for expedience.